cmd/skywire-updater/commands/gen-config.go

```diff
@@ -6,11 +6,10 @@ import (
 	"os"
 	"path/filepath"
 
-	"github.com/spf13/cobra"
-	"gopkg.in/yaml.v2"
-
 	"github.com/skycoin/skywire-updater/internal/pathutil"
 	"github.com/skycoin/skywire-updater/pkg/update"
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v2"
```

Comment:
This does not look `goimported`?

Reply:
Hi Evan. Can you clarify a bit more here? Do you mean that these imports shouldn't be there? I think it could be because I am having issue with my goimported. golangcli-lint installation fails for me whenever I run make install-linters. I am not able to narrow down the reason at the moment but will investigate further. Thanks!

Reply:
@atang152 have you tried installing with env `GO111MODULE=off`?

Reply:
@atang152 to format the code:

```bash
$ goimports -w -local github.com/skycoin/skywire-updater ./cmd
$  goimports -w -local github.com/skycoin/skywire-updater ./pkg
```

(Using https://github.com/skycoin/skywire/blob/c3a15be793212133c74259cda857fbfa9efb7df2/Makefile#L85-L88 as reference.)

Reply:
The `Makefile` is outdated (as you know) and doesn't format the `/cmd` folder.